Wikipedia/Code/ExploreViewController.swift

```diff
@@ -927,7 +905,71 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
     }
 
     var addArticlesToReadingListVCDidDisappear: (() -> Void)? = nil
-    
+
+    // TODO: Remove after expiry date (1 March 2025)
+    private func presentYearInReviewAnnouncement() {
+
+        if UIDevice.current.userInterfaceIdiom == .pad && navigationBar.hiddenHeight > 0 {
+            return
+        }
+
+        let languages = ["fr", "it"]
+        guard YearInReviewFeatureAnnouncementTimeBox.isAnnouncementActive() else {
+            return
+        }
+
+        guard let appLanguage = dataStore.languageLinkController.appLanguage else {
+            return
+        }
+
+        guard languages.contains(appLanguage.languageCode) else {
```

Comment:
I think `shouldShowYearInReviewEntryPoint` below should already cover checking for this.